### DISCUSSION TITLE
Can I force ENS resolver to be always checking against ethereum node

### DISCUSSION THREAD

**Discussion by grmkris** [rainbow-me/rainbowkit#481] [Q&A] [answered]
<img width="518" alt="image" src="https://user-images.githubusercontent.com/15545195/172478663-c12a14a1-6112-4565-bb81-99fa8c292b07.png">

<img width="643" alt="image" src="https://user-images.githubusercontent.com/15545195/172478784-251cd3f3-1e0a-489c-9bf1-363ae1b13604.png">

Since we are using WAGMI under the hood I don't believe this should be hard. Example with wagmi hook:
```
  const { data: envAvatar, isLoading: isLoadingEnsAvatar } = useEnsAvatar({
    addressOrName: props.comment.owner.id,
    chainId: 1,
  });
  const { data: ensName, isLoading: isLoadingEnsName } = useEnsName({
    address: props.comment.owner.id,
    chainId: 1,
  });
```

If you can show me place in code to do this, I can also make the change and post a PR to you

**Comment by markdalgleish** [comment, marked as answer]
This was fixed in [RainbowKit v0.4.1.](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.4.1)
